src/imageTools/magnify.js

```diff
@@ -111,13 +112,7 @@ function drawMagnificationTool (eventData) {
   // The 'not' magnifyTool class here is necessary because cornerstone places
   // No classes of it's own on the canvas we want to select
   const canvas = element.querySelector('canvas:not(.magnifyTool)');
-  const context = canvas.getContext('2d');
-
-  context.setTransform(1, 0, 0, 1, 0, 0);
-
-  const zoomCtx = magnifyCanvas.getContext('2d');
-
-  zoomCtx.setTransform(1, 0, 0, 1, 0, 0);
+  const zoomCtx = getNewContext(magnifyCanvas);
```

Comment:
I'm just doing a quick pass over. Were you able to drop `context` because it's not used? I was under the impression that the magnify tool used two separate context's for rendering (that may have been the old version)

Reply:
Yep, the other `context` is unused so can be safely removed.